### ISSUE TITLE
Add support for accepting compressed items/other transfer syntaxes.

### ISSUE THREAD

**Issue by richrosenbaum** [mercure-imaging/mercure#25]
I sent JPEG-LS lossless, JPEG 2000 lossless and JPEG lossless studies to the router, and the response was "transfer syntax not supported", and required the server to decompress before sending to the router which is not optimal.

dcmtk storescp has an option to accept all supported transfer syntaxes which would be a great addition.

Also, I think it would be a good idea to document the current configured method (that mercure does not support compression at this time) in the documentation. If/when you implement accepting all transfer syntaxes documentation can be changed to match the current status.

Finally, maybe change the dicom items from uncompressed to compressed before sending for example. You would probably have to rewrite the file. It's just a thought, but I think it would be a good roadmap item.

**Comment by tblock79** [comment]
We will come up with a concept how to best support compressed DICOM files. Currently, the DICOM sender is asked to transfer the images in decompressed format (which senders are required to be able to per DICOM spec).

In the meantime, acceptance of compressed DICOMs can be enabled by adding "--accept-all" to the storescp call in the last line of the file "receiver.sh"

**Comment by tblock79** [comment]
Added configuration option to enable acceptance of "any transfer syntax". This is not enabled by default to improve compatibility (and to ease development of mercure processing modules). However, as I can see that some users might want to avoid decompression of incoming studies (if using mercure solely as router), we added this option. A warning will be shown that the option should be used carefully.
